python/llm/src/bigdl/llm/transformers/low_bit_linear.py

```diff
@@ -388,11 +388,20 @@ def forward(self, x: torch.Tensor):
             if IS_SERVER and (not IS_SPR) and \
                     self.qtype == SYM_INT4 and x_2d.shape[0] >= TORCH_LINEAR_THRESHOLD:
                 x0_fp32 = ggml_int4_convert_fp32(x0, self.weight_shape, self.weight_length)
+                # Parallel F.linear should be avoided, thus deepspeed allreduce after the operation
```

Comment:
These lines are a patch to #9230, which gets rid of redundant codes to avoid conflicts

Reply:
Remove TODO since we added implementation for CPU